src/common/types/data_chunk.cpp

```diff
@@ -13,6 +13,7 @@
 #include "duckdb/common/arrow.hpp"
 #include "duckdb/common/vector.hpp"
 #include "duckdb/common/to_string.hpp"
+#include <array>
```

Comment:
Can we move this to "duckdb/common/array.hpp", similar to e.g. "duckdb/common/unordered_map.hpp", so we can avoid the std:: prefix and limit standard library includes to a single location?